tailwind.config.js

```diff
@@ -10,6 +10,13 @@ module.exports = {
     extend: {
       screens: {
         fold: "300px",
+        '1.5xl': '1500px',
+        '2.5xl': '1900px',
+        '3xl': '2500px',
+        '4xl': '3000px',
+        '5xl': '3500px',
+        '6xl': '4000px',
+        '7xl': '4500px',
```

Comment:
What is the reason behind adding all these screen size?

Reply:
They're for gradual grid column changing depending on screen size. I was experimenting with increasing the maximum screen size because my resolution is somewhat common. The 1.5 and 2.5 are just the specific sizes necessary for the pathway grid to render without overlapping cells, if there's a better way to do this you can lmk.

Reply:
Since we are limiting the overall screen size to xl (iirc), I don't see any reason to make the inner content (e.g. search course page) larger than it need to be.

Reply:
I think I limited it to something larger, xl looked bad on my monitor. Some of those display sizes aren't used though yeah.